src/service/performance-impl.js

```diff
@@ -398,13 +398,7 @@ export class Performance {
     }
 
     if (this.supportsEventTimingAPIv77_) {
-      // It's preferred to read first input delay entries that already occurred
-      // through the `buffered: true` flag, so create a separate
-      // PerformanceObserver to read this metric.
-      const firstInputObserver = new this.win.PerformanceObserver(list => {
-        list.getEntries().forEach(processEntry);
-        this.flush();
-      });
+      const firstInputObserver = this.createPerformanceObserver_();
```

Comment:
Why does this observer not receive `processEntry` as a parameter?

Reply:
Fixed in bbde314258d6ba53c166642804f035bcf6af364f